src/components/Card/Card.driver.ts

```diff
@@ -0,0 +1,22 @@
+import { StylableDOMUtil } from '@stylable/dom-test-kit';
+import style from './Card.st.css';
+
+export const cardDriverFactory = ({ element }) => {
+  const stylableDOMUtil = new StylableDOMUtil(style);
+
+  const getImageContainerElement = () =>
+    element.querySelector(stylableDOMUtil.scopeSelector('.mediaContainer'));
+  const getInfoContainerElement = () =>
+    element.querySelector(stylableDOMUtil.scopeSelector('.infoContainer'));
+
+  return {
+    exists: () => !!element,
+    getMediaContent: () => getImageContainerElement().innerHTML,
```

Comment:
instead of returning the html - return the dom node.
if you're migrating to unidriver - it should return the `getNative` element